### ISSUE TITLE
Card design

### ISSUE THREAD

**Issue by digambar-t7** [yessGlory17/nft-marketplace#24]
Hey bro, 
              I think that the card design needs to be modified. Observing other trending marketplaces, I came to following improvements:
1. Price should be changed to bid like last bid price. Because nfts are auctioned & not sold at any fixed price.
2. Buy now should not be directly on the cards. Instead, when a user feels interested in the card he must click on the card. This will take him to that card-specific-webpage. This will allow us to put all the info,comments/opinions of others of that nft including the price.
3. Nfts/cards should be arranged category-
 
Also, I didn't get what you exactly mean by dark-nft & nft-hunting. Please explain.

**Comment by yessGlory17** [comment]
You are right. However, for this, we must first create a new card design. If you have made a card design, I would like to review it and tell you my opinion.
